### PR TITLE
Grab historial feeds

### PR DESCRIPTION
**Pull request by urfolomeus**
This is a PR to fetch all the blog content. We have to it this way for the blog because it is paginated to only return 10 records at a time.

### CODE REVIEW

**Comment by darokel** [comment]
:+1:

**Comment by neoeno** [comment]
:+1:

File: spec/fixtures/blog/many_items.rss [hunk 1]
```diff
@@ -109,10 +109,10 @@
 
 </description>
       <pubDate>Wed, 29 Apr 2015 16:19:23 +0000</pubDate>
-      <guid isPermaLink="false">urn:uuid:5d7e0615-9b81-4083-827e-050fe916dc5f</guid>
-      <comments>http://blog.moneyadviceservice.org.uk/are-you-raiding-your-savings#comments</comments>
+      <guid isPermaLink="false">urn:uuid:5d7e0615-9b81-4083-827e-050fe916dc56</guid>
+      <comments>http://blog.moneyadviceservice.org.uk/are-you-raiding-my-savings#comments</comments>
       <category>Savings</category>
-      <link>http://blog.moneyadviceservice.org.uk/are-you-raiding-your-savings</link>
+      <link>http://blog.moneyadviceservice.org.uk/are-you-raiding-my-savings</link>
```
Comment (paulanthonywilson) [inline spec/fixtures/blog/many_items.rss]: Hands off my savings!

**Comment by paulanthonywilson** [comment]
:+1:

**Pull request merged**